libraries/AP_HAL_Linux/Util_RPI.cpp

```diff
@@ -49,10 +49,6 @@ int UtilRPI::_check_rpi_version_by_rev()
             
             _rpi_version = cpu;
             
-            if (_rpi_version==0) {
```

Comment:
I believe this was some leftover code from trying to treat boards according to the board name(0/1/2/3/4, where we want pi0 and pi1 to be the same because they have the same soc) vs the soc ID or soc ranges(where remapping pi0 to pi1 is unnecessary since the soc ID or ranges would be the same for those boards already.